tests/acceptance/course-admin/view-diffs-test.js

```diff
@@ -32,66 +31,11 @@ module('Acceptance | course-admin | view-diffs', function (hooks) {
         filename: 'server.rb',
         diff: `    end
 
-    def listen
+    def taste
```

Comment:
@VasylMarchuk can we stick to the original diff here? Or something more realistic? 

This isn't close to what we'd see on a real-life submission: 


<img width="411" alt="Screenshot 2024-09-22 at 14 40 34" src="https://github.com/user-attachments/assets/924cb060-1a05-4fca-950e-6e3bd509ae98">